healthtools_ke_api/nurses.py

```diff
@@ -1,31 +1,31 @@
-import requests
-
 from bs4 import BeautifulSoup
-from flask import Blueprint, request, jsonify, make_response, json
-from werkzeug.exceptions import HTTPException, default_exceptions
-from memcache import Client
+from flask import Blueprint, request, jsonify, make_response, json, current_app
 
-from healthtools_ke_api.config import *
 from healthtools_ke_api.analytics import track_event
+from healthtools_ke_api.settings import MEMCACHED_URL
+
+import requests
+import memcache
+
 
 nurses_api = Blueprint('nurses_api', __name__)
-nurse_fields = ["name", "licence_no", "valid_till"]
-cache = Client([(MEMCACHED_URL)], debug=True)
+cache = memcache.Client([(MEMCACHED_URL)], debug=True)  # cache server
 
-NURSING_COUNCIL_URL = 'http://nckenya.com/services/search.php?p=1&s={}'
+nurse_fields = ["name", "licence_no", "valid_till"]
+NURSING_COUNCIL_URL = "http://nckenya.com/services/search.php?p=1&s={}"
 
 
 @nurses_api.route('/', methods=['GET'])
-def index():
+def home():
```

Comment:
`index` seems more best practice instead of `home` which looks to display a view.